quarkus-integration-test-liquibase-mongodb fails in native with latest GraalVM 22.3-dev build

### Describe the bug

`liquibase-mongodb` quarkus integration test fails with latest GraalVM CE dev build in native mode with:

```
========================================================================================================================
GraalVM Native Image: Generating 'quarkus-integration-test-liquibase-mongodb-999-SNAPSHOT-runner' (executable)...
========================================================================================================================
[1/7] Initializing...                                                                                    (8.8s @ 0.32GB)
 Version info: 'GraalVM 22.3.0-dev Java 17 CE'
 Java version info: '17.0.4+7-jvmci-22.3-b02'
 C compiler: gcc (redhat, x86_64, 12.1.1)
 Garbage collector: Serial GC
 4 user-specific feature(s)
 - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
 - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.jboss.threads] categories
 - io.quarkus.runtime.graal.ResourcesFeature: Register each line in META-INF/quarkus-native-resources.txt as a resource on Substrate VM
 - org.graalvm.home.HomeFinderFeature: Finds GraalVM paths and its version number
[2/7] Performing analysis...  [*]                                                                       (12.7s @ 0.71GB)
   5,922 (72.64%) of  8,152 classes reachable
   5,782 (54.58%) of 10,594 fields reachable
  19,978 (33.01%) of 60,516 methods reachable
     783 classes,   235 fields, and 5,677 methods registered for reflection

10 fatal errors detected:
Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing liquibase.Scope.getCurrentScope() 
Parsing context:
   at liquibase.Scope.getCurrentScope(Scope.java:71)
   at liquibase.lockservice.StandardLockService.isDatabaseChangeLogLockTableInitialized(StandardLockService.java:192)
   at liquibase.lockservice.StandardLockService.init(StandardLockService.java:130)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:286)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)
   at liquibase.lockservice.StandardLockService.acquireLock(StandardLockService.java:276)

	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultStaticInvokeTypeFlow.update(DefaultStaticInvokeTypeFlow.java:64)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:492)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Discovered a type for which getDeclaringClass0 cannot be called: liquibase.osgi.Activator$OSGIContainerChecker. This error is reported at image build time because class liquibase.osgi.Activator$OSGIContainerChecker is registered for linking at image build time by command line
	at parsing liquibase.Scope.getCurrentScope(Scope.java:100)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3379)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3331)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3176)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:84)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:261)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:684)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:168)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:343)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
	... 12 more
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Discovered a type for which getDeclaringClass0 cannot be called: liquibase.osgi.Activator$OSGIContainerChecker. This error is reported at image build time because class liquibase.osgi.Activator$OSGIContainerChecker is registered for linking at image build time by command line
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.handleLinkageError(SVMHost.java:423)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.getDeclaringClass(SVMHost.java:409)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.createHub(SVMHost.java:397)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.registerType(SVMHost.java:243)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.createType(AnalysisUniverse.java:302)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:212)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookup(AnalysisUniverse.java:189)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.<init>(AnalysisMethod.java:128)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.PointsToAnalysisMethod.<init>(PointsToAnalysisMethod.java:62)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.PointsToAnalysisFactory.createMethod(PointsToAnalysisFactory.java:35)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.createMethod(AnalysisUniverse.java:429)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:417)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.lookupMethod(WrappedConstantPool.java:170)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupMethodInPool(BytecodeParser.java:4191)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.lookupMethodInPool(SharedGraphBuilderPhase.java:138)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupMethod(BytecodeParser.java:4185)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1648)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5236)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3371)
	... 27 more
Caused by: java.lang.NoClassDefFoundError: org/osgi/framework/BundleActivator
	at java.base/java.lang.Class.getDeclaringClass0(Native Method)
	at java.base/jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.getDeclaringClass(SVMHost.java:406)
	... 44 more
Caused by: java.lang.ClassNotFoundException: org.osgi.framework.BundleActivator
	... 49 more
```

### Expected behavior

Native image build works.

### Actual behavior

Native image build fails with: `java.lang.ClassNotFoundException: org.osgi.framework.BundleActivator`

### How to Reproduce?

Steps to reproduce:

1. Download the latest 22.3.0-dev GraalVM CE build. For example from here: https://github.com/graalvm/graalvm-ce-dev-builds/releases/tag/22.3.0-dev-20220805_1946
2. Use it as `GRAALVM_HOME`
3. Run the `liquibase-mongodb` integration test:
```
pushd integration-tests
../mvnw --fae -pl 'liquibase-mongodb' -Dnative -Dtest-containers -Dmaven.test.failure.ignore=true -Dstart-containers -Dnative.surefire.skip -Dformat.skip -Dno-descriptor-tests  -Dlog.level=ALL clean verify
popd
```

### Output of `uname -a` or `ver`

_No response_

### Output of `java -version`

11 or 17 based GraalVM (doesn't matter)

### GraalVM version (if different from Java)

22.3.0-dev

### Quarkus version or git rev

dc51ebc7ea34339b43139cd16bf8ac4d44ef868a

### Build tool (ie. output of `mvnw --version` or `gradlew --version`)

_No response_

### Additional information

This seems to be related to this GraalVM change:
https://github.com/oracle/graal/commit/ce3d101fba368cdb4bd8eebad770e5c59c8aa57a

A similar issue with `redis-client` IT is: https://github.com/quarkusio/quarkus/issues/27211

This is also reproducible with just `liquibase` (no mongodb):

```
========================================================================================================================
GraalVM Native Image: Generating 'quarkus-integration-test-liquibase-999-SNAPSHOT-runner' (executable)...
========================================================================================================================
[1/7] Initializing...                                                                                    (7.5s @ 0.31GB)
 Version info: 'GraalVM 22.3.0-dev Java 17 CE'
 Java version info: '17.0.4+7-jvmci-22.3-b02'
 C compiler: gcc (redhat, x86_64, 12.1.1)
 Garbage collector: Serial GC
 4 user-specific feature(s)
 - io.quarkus.runner.Feature: Auto-generated class by Quarkus from the existing extensions
 - io.quarkus.runtime.graal.DisableLoggingFeature: Disables INFO logging during the analysis phase for the [org.jboss.threads] categories
 - io.quarkus.runtime.graal.ResourcesFeature: Register each line in META-INF/quarkus-native-resources.txt as a resource on Substrate VM
 - org.graalvm.home.HomeFinderFeature: Finds GraalVM paths and its version number
[2/7] Performing analysis...  [18:31:34,621 INFO  [org.jbo.res.res.i18n] RESTEASY002225: Deploying javax.ws.rs.core.Application: class io.quarkus.it.liquibase.LiquibaseApp
*]                                                                       (10.9s @ 1.66GB)
   5,766 (73.33%) of  7,863 classes reachable
   6,065 (55.41%) of 10,945 fields reachable
  19,488 (33.24%) of 58,635 methods reachable
     770 classes,   458 fields, and 5,119 methods registered for reflection

2 fatal errors detected:
Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing liquibase.Scope.getCurrentScope() 
Parsing context:
   at liquibase.Scope.getCurrentScope(Scope.java:71)
   at liquibase.command.CommandArgumentDefinition$Building.build(CommandArgumentDefinition.java:259)
   at liquibase.command.core.ChangelogSyncCommandStep.<clinit>(ChangelogSyncCommandStep.java:35)

	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultStaticInvokeTypeFlow.update(DefaultStaticInvokeTypeFlow.java:64)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:492)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Discovered a type for which getDeclaringClass0 cannot be called: liquibase.osgi.Activator$OSGIContainerChecker. This error is reported at image build time because class liquibase.osgi.Activator$OSGIContainerChecker is registered for linking at image build time by command line
	at parsing liquibase.Scope.getCurrentScope(Scope.java:100)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3379)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3331)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3176)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:84)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:261)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:684)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:168)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:343)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
	... 12 more
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Discovered a type for which getDeclaringClass0 cannot be called: liquibase.osgi.Activator$OSGIContainerChecker. This error is reported at image build time because class liquibase.osgi.Activator$OSGIContainerChecker is registered for linking at image build time by command line
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.handleLinkageError(SVMHost.java:423)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.getDeclaringClass(SVMHost.java:409)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.createHub(SVMHost.java:397)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.registerType(SVMHost.java:243)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.createType(AnalysisUniverse.java:302)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:212)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookup(AnalysisUniverse.java:189)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.<init>(AnalysisMethod.java:128)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.PointsToAnalysisMethod.<init>(PointsToAnalysisMethod.java:62)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.PointsToAnalysisFactory.createMethod(PointsToAnalysisFactory.java:35)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.createMethod(AnalysisUniverse.java:429)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:417)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.lookupMethod(WrappedConstantPool.java:170)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupMethodInPool(BytecodeParser.java:4191)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.lookupMethodInPool(SharedGraphBuilderPhase.java:138)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupMethod(BytecodeParser.java:4185)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1648)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5236)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3371)
	... 27 more
Caused by: java.lang.NoClassDefFoundError: org/osgi/framework/BundleActivator
	at java.base/java.lang.Class.getDeclaringClass0(Native Method)
	at java.base/jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.getDeclaringClass(SVMHost.java:406)
	... 44 more
Caused by: java.lang.ClassNotFoundException: org.osgi.framework.BundleActivator
	... 49 more
Fatal error: com.oracle.graal.pointsto.util.AnalysisError$ParsingError: Error encountered while parsing liquibase.Scope.getCurrentScope() 
Parsing context:
   at liquibase.Scope.getCurrentScope(Scope.java:71)
   at liquibase.command.CommandArgumentDefinition$Building.build(CommandArgumentDefinition.java:257)
   at liquibase.command.core.ChangelogSyncCommandStep.<clinit>(ChangelogSyncCommandStep.java:35)

	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.parsingError(AnalysisError.java:153)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:104)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.ensureFlowsGraphCreated(MethodTypeFlow.java:83)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.getOrCreateMethodFlowsGraph(MethodTypeFlow.java:65)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.typestate.DefaultStaticInvokeTypeFlow.update(DefaultStaticInvokeTypeFlow.java:64)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.PointsToAnalysis$1.run(PointsToAnalysis.java:492)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.executeCommand(CompletionExecutor.java:193)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.CompletionExecutor.lambda$executeService$0(CompletionExecutor.java:177)
	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: com.oracle.graal.pointsto.util.AnalysisError: parsing had failed in another thread
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.util.AnalysisError.shouldNotReachHere(AnalysisError.java:177)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:719)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.parse(MethodTypeFlowBuilder.java:168)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlowBuilder.apply(MethodTypeFlowBuilder.java:343)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.MethodTypeFlow.createFlowsGraph(MethodTypeFlow.java:93)
	... 12 more
Caused by: org.graalvm.compiler.java.BytecodeParser$BytecodeParserError: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Discovered a type for which getDeclaringClass0 cannot be called: liquibase.osgi.Activator$OSGIContainerChecker. This error is reported at image build time because class liquibase.osgi.Activator$OSGIContainerChecker is registered for linking at image build time by command line
	at parsing liquibase.Scope.getCurrentScope(Scope.java:100)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.throwParserError(BytecodeParser.java:2518)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.throwParserError(SharedGraphBuilderPhase.java:110)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3379)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.handleBytecodeBlock(BytecodeParser.java:3331)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBlock(BytecodeParser.java:3176)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.build(BytecodeParser.java:1138)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.buildRootMethod(BytecodeParser.java:1030)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.GraphBuilderPhase$Instance.run(GraphBuilderPhase.java:84)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase.run(SharedGraphBuilderPhase.java:84)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.run(Phase.java:49)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.BasePhase.apply(BasePhase.java:261)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:42)
	at jdk.internal.vm.compiler/org.graalvm.compiler.phases.Phase.apply(Phase.java:38)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.flow.AnalysisParsedGraph.parseBytecode(AnalysisParsedGraph.java:135)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.ensureGraphParsed(AnalysisMethod.java:684)
	... 15 more
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Discovered a type for which getDeclaringClass0 cannot be called: liquibase.osgi.Activator$OSGIContainerChecker. This error is reported at image build time because class liquibase.osgi.Activator$OSGIContainerChecker is registered for linking at image build time by command line
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.handleLinkageError(SVMHost.java:423)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.getDeclaringClass(SVMHost.java:409)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.createHub(SVMHost.java:397)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.registerType(SVMHost.java:243)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.createType(AnalysisUniverse.java:302)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:212)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookup(AnalysisUniverse.java:189)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisMethod.<init>(AnalysisMethod.java:128)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.PointsToAnalysisMethod.<init>(PointsToAnalysisMethod.java:62)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.PointsToAnalysisFactory.createMethod(PointsToAnalysisFactory.java:35)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.createMethod(AnalysisUniverse.java:429)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.meta.AnalysisUniverse.lookupAllowUnresolved(AnalysisUniverse.java:417)
	at org.graalvm.nativeimage.pointsto/com.oracle.graal.pointsto.infrastructure.WrappedConstantPool.lookupMethod(WrappedConstantPool.java:170)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupMethodInPool(BytecodeParser.java:4191)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.phases.SharedGraphBuilderPhase$SharedBytecodeParser.lookupMethodInPool(SharedGraphBuilderPhase.java:138)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.lookupMethod(BytecodeParser.java:4185)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.genInvokeStatic(BytecodeParser.java:1648)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.processBytecode(BytecodeParser.java:5236)
	at jdk.internal.vm.compiler/org.graalvm.compiler.java.BytecodeParser.iterateBytecodesForBlock(BytecodeParser.java:3371)
	... 27 more
Caused by: java.lang.NoClassDefFoundError: org/osgi/framework/BundleActivator
	at java.base/java.lang.Class.getDeclaringClass0(Native Method)
	at java.base/jdk.internal.reflect.GeneratedMethodAccessor10.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.graalvm.nativeimage.builder/com.oracle.svm.hosted.SVMHost.getDeclaringClass(SVMHost.java:406)
	... 44 more
Caused by: java.lang.ClassNotFoundException: org.osgi.framework.BundleActivator
	... 49 more
------------------------------------------------------------------------------------------------------------------------
                        0.6s (3.2% of total time) in 15 GCs | Peak RSS: 2.17GB | CPU load: 4.95
========================================================================================================================
Failed generating 'quarkus-integration-test-liquibase-999-SNAPSHOT-runner' after 18.8s.
Error: Image build request failed with exit status 1
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  33.517 s
[INFO] Finished at: 2022-08-11T18:31:43+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:999-SNAPSHOT:build (default) on project quarkus-integration-test-liquibase: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkus.deployment.pkg.steps.NativeImageBuildStep#build threw an exception: io.quarkus.deployment.pkg.steps.NativeImageBuildStep$ImageGenerationFailureException: Image generation failed. Exit code: 1
[ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.imageGenerationFailed(NativeImageBuildStep.java:420)
[ERROR] 	at io.quarkus.deployment.pkg.steps.NativeImageBuildStep.build(NativeImageBuildStep.java:261)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
[ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:833)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

--- a/bom/application/pom.xml
+++ b/bom/application/pom.xml
@@ -5395,6 +5395,12 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <!-- liquibase-core has a dependency on org.osgi:osgi.core -->
+            <dependency>
+                <groupId>org.osgi</groupId>
+                <artifactId>osgi.core</artifactId>
+                <version>${osgi.version}</version>
+            </dependency>
             <dependency>
                 <groupId>org.liquibase.ext</groupId>
                 <artifactId>liquibase-mongodb</artifactId>

--- a/extensions/liquibase-mongodb/runtime/pom.xml
+++ b/extensions/liquibase-mongodb/runtime/pom.xml
@@ -26,6 +26,10 @@
             <artifactId>liquibase-core</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>osgi.core</artifactId>
+        </dependency>
+        <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
         </dependency>

--- a/extensions/liquibase/runtime/pom.xml
+++ b/extensions/liquibase/runtime/pom.xml
@@ -22,6 +22,10 @@
             <artifactId>liquibase-core</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>osgi.core</artifactId>
+        </dependency>
+        <dependency>
             <groupId>org.yaml</groupId>
             <artifactId>snakeyaml</artifactId>
         </dependency>

